src/sync/discovery.ts

```diff
@@ -5,12 +5,6 @@ import { Application } from "probot";
 import { SyncStatus } from "../models/subscription";
 import { getLogger } from "../config/logger";
 
-const jobOpts = {
-	removeOnComplete: true,
-	removeOnFail: true,
-	attempts: 3
-};
```

Comment:
So we removing it from here because we put these settings globally on the queue, right?

Reply:
yes